Sources/Cache/CacheApi.php

```diff
@@ -193,7 +50,7 @@ abstract class cache_api implements cache_api_interface
 	 */
 	public function __construct()
 	{
-		$this->setPrefix();
+		$this->setPrefix('');
```

Comment:
Is it necessary to send a parameter?

Reply:
nope, was added by the linting script, will remove.